### PR TITLE
Add Mark All as Read

### LINKED ISSUE #151: Add option to mark all messages read in a room
**Actual Behaviour**
No option to mark read messages in a single click

**Expected Behaviour**
Should have an option to mark read all messages

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Fixes #151

**Changes:**
Added Option to Mark Read All messages
**Screenshots for the change:**
screenshot-1533218988998

### CODE REVIEW

File: app/src/main/java/com/codingblocks/chatter/RoomActivity.java [hunk 1]
```diff
@@ -146,9 +149,42 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+                markRed(roomId);
+                break;
 
         }
         return super.onOptionsItemSelected(item);
     }
 
+    private void markRed(String roomId) {
+        RequestBody requestBody = new FormBody.Builder()
+                .add("favourite", "true")
```
Comment (abishekvashok) [inline app/src/main/java/com/codingblocks/chatter/RoomActivity.java]: Why are you adding this?

**Comment by aggarwalpulkit596** [comment]
It was added by mistake will remove it

**Comment by aggarwalpulkit596** [comment]
@championswimmer resolved conflicts

**Comment by aggarwalpulkit596** [comment]
@the-dagger fixed conflicts

**Comment by the-dagger** [comment]
I've restarted the build, let's see

**Comment by aggarwalpulkit596** [comment]
Same error

**Comment by aggarwalpulkit596** [comment]
@the-dagger it is now passed.

**Pull request merged**